aitinero/src/app/home/location/page.tsx

```diff
@@ -25,7 +25,9 @@ export default function LocationForm() {
                 <p><strong>Where</strong> do you want to go?</p>
                 </div>
                 <div className="my-2">
-                    <Input type="text" className="rounded-md p-2 w-96" {...register('location')}/>
+                    <Input type="text" className="rounded-md p-2 w-96" {...register('location',
+                        {validate: (value) => value.length > 0 && !!value.match(/^[a-zA-Z ]*$/)}
```

Comment:
```suggestion
                        {validate: (value) => value.length > 0 && /^[a-zA-Z ]*$/.test(value)}
```
I suggest using `.test` since it returns a boolean, and it's much faster.
Source: https://stackoverflow.com/questions/10940137/regex-test-v-s-string-match-to-know-if-a-string-matches-a-regular-expression